lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -88,7 +88,10 @@ class TraceOfTab extends ComputedArtifact {
       firstMeaningfulPaint = lastCandidate;
     }
 
-    const onLoad = keyEvents.find(e => e.name === 'MarkLoad' && e.ts > navigationStart.ts);
+    const onLoad = frameEvents.find(e => e.name === 'loadEventStart' && e.ts > navigationStart.ts);
```

Comment:
@paulirish `MarkLoad` was not onload in most cases I saw during my investigations, what's supposed to be the difference between it and loadEventStart?